api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -122,6 +122,14 @@ message ExtAuthz {
   // :ref:`certificate<envoy_v3_api_field_service.auth.v3.AttributeContext.Peer.certificate>`.
   bool include_peer_certificate = 10;
 
+  // Specifies whether HTTP status codes for gRPC calls are passed intact from
```

Comment:
Can you elaborate on when this is used? Usually an ext_authz gRPC service will return 200, so what scenario does this matter? I'm asking for the reader, so updating the comments here makes sense; I personally am familiar with when gRPC can return non-200 (e.g. early failure), but less clear to docs reader.